packages/transactions/src/builders.ts

```diff
@@ -137,6 +140,78 @@ export async function estimateTransfer(
   return feeRate * txBytes;
 }
 
+interface FeeEstimateResponse {
+  cost_scalar_change_by_byte: bigint,
+  estimated_cost: {
+    read_count: bigint,
+    read_length: bigint,
+    runtime: bigint,
+    write_count: bigint,
+    write_length: bigint
+  },
+  estimated_cost_scalar: bigint,
+  estimated_fee_rates: {
+    high: bigint,
+    low: bigint,
+    middle: bigint
+  },
+  estimated_fees: FeeEstimate
+}
+
+interface FeeEstimate {
+  high: bigint,
+  middle: bigint,
+  low: bigint,
+}
```

Comment:
Isn't the endpoint an array, not an object with high/middle/low?

Reply:
not according to this PR description:

https://github.com/blockstack/stacks-blockchain/pull/2872